### PR TITLE
feat: Sorting the list of tokens to send

### LINKED ISSUE #2046: Change default token for sending
### Current behavior
When a user holds multiple tokens, and they go to Send, the selected token by default is the one with the highest balance. This could be any token, like MOBI, WETH, etc. 

### Desired behavior
The P2P sending/requesting use case (e.g. users paying each other back for coffee) is most obvious using stablecoins.  The app should set the default token for sending/requesting to be the highest stablecoin balance 

### Steps to Reproduce
Home > Send > select contact 

### Versions
All

### Eng
- [ ] in the send flow token list, the order of tokens displayed should be celo native stable tokens (ordered by balance) -> CELO -> all other tokens (ordered by balance)
- [ ] the default token in the send flow should be the first token on the token list after the above change (i.e. the celo native stable token with the highest balance if it exists)

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

Change in the sorting logic for the list of tokens to send, to show first stable tokens then celo and then others tokens (all sorted by user usd balance in each "category").

### Tested

Locally in an iOS emulator and unit tests

### How others should test

You should see the new sorted list in the send flow.

### Related issues

- Fixes #2046

### Screenshot
Simulator Screen Shot - iPhone 12 - 2022-03-11 at 13 25 54

### CODE REVIEW

File: packages/mobile/src/tokens/utils.ts [hunk 1]
```diff
@@ -30,3 +30,55 @@ export function sortByUsdBalance(token1: TokenBalance, token2: TokenBalance) {
...
   return token?.isCoreToken && token.symbol !== 'CELO'
 }
+
+/**
+ * It sorts in 3 categories:
+ * Stable tokens: (cUSD, cEUR, cREAL) (sorted by usd balance)
+ * CELO
+ * Other tokens: sorted by usd balance
+ *
+ * If someone comes with a better name for this, I would appreciate it.
```
Comment (gnardini) [inline packages/mobile/src/tokens/utils.ts:40]: I really like the name btw, it's very descriptive, I don't think we can do better than that

**Review by gnardini — APPROVED** [review]
Great job! 🔥

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Test Flight Release build V1.29.0, Android Internal Release Build V1.29.0 & updated status in test rails.

**Test rail link:**

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified above task on Latest Android Internal Release Build V1.29.0 & updated status in test rails.

- User is able to see highest balance tokens on first position firstly Stable coins, CELO & other tokens in Multitoken Popup on Enter Amount screen

**Test rail link:**
**Android 12:** https://valoraapp.testrail.io/index.php?/tests/view/20978 

**Devices:** Google Pixel 4a (12.0)
cc: @dievazqu 
Thanks..!!

**Pull request merged**